backend/app/api/endpoints/projects.py

```
@@ -0,0 +1,241 @@
+):
+    """
+    Create a new project.
+    The current user becomes the project owner.
+    """
+    try:
+        return project_service.create_project(
+            db=db, project_data=project_create, user_id=current_user.id
+        )
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)